client/src/app/site/users/components/user-list/user-list.component.ts

```diff
@@ -441,7 +423,41 @@ export class UserListComponent extends BaseListViewComponent<ViewUser> implement
             this.operator.hasPerms(Permission.usersCanManage) ||
             (this.allowSelfSetPresent && this.operator.operatorId === viewUser.id);
         if (isAllowed) {
-            this.repo.setPresent(viewUser, !this.isUserPresent(viewUser));
+            this.repo.setPresent(!this.isUserPresent(viewUser), viewUser);
+        }
+    }
+
+    /**
+     * Handler for bulk setting/unsetting the 'active' attribute.
+     * Uses selectedRows defined via multiSelect mode.
+     */
+    private async setStateSelected(field: UserStateField): Promise<void> {
+        let options: [string, string];
+        let verboseStateName: string;
+        switch (field) {
+            case 'is_active':
+                options = [_('active'), _('inactive')];
+                verboseStateName = 'active';
+                break;
+            case 'is_present_in_meetings':
+                options = [_('present'), _('absent')];
+                verboseStateName = 'present';
+                break;
+            case 'is_physical_person':
+                options = [_('natural person'), _('no natural person')]; // switched order: no committee=physical person
+                verboseStateName = 'committee';
```

Comment:
can you also rename `committee` here, so all logical connections to OS3 are gone?

Reply:
I just removed the `verboseStateName` since it is never used.